internal/data/db_sql.go

```diff
@@ -186,7 +191,12 @@ const sqlFmtFeature = "SELECT %v %v FROM \"%s\".\"%s\" WHERE \"%v\" = $1 LIMIT 1
 
 func sqlFeature(tbl *Table, param *QueryParam) string {
 	geomCol := sqlGeomCol(tbl.GeometryColumn, tbl.Srid, param)
-	propCols := sqlColList(param.Columns, tbl.DbTypes, true)
+
+	tblTypes := make(map[string]string)
```

Comment:
Avoid copy here too?  Or at least move this code into a function so it can be shared.